plugin/trino-hive/src/main/java/io/trino/plugin/hive/fs/CachingDirectoryLister.java

```diff
@@ -65,6 +65,10 @@ public CachingDirectoryLister(HiveConfig hiveClientConfig)
 
     public CachingDirectoryLister(Duration expireAfterWrite, DataSize maxSize, List<String> tables, Predicate<FileEntry> filterPredicate)
     {
+        requireNonNull(expireAfterWrite);
+        requireNonNull(maxSize);
+        requireNonNull(tables);
+        requireNonNull(filterPredicate);
```

Comment:
Can we have a meaningfull message for rnn ?

Reply:
fixed